packages/moonstream-components/src/core/providers/Web3Provider/index.tsx

```diff
@@ -334,6 +337,7 @@ const Web3Provider = ({ children }: { children: JSX.Element }) => {
     <Web3Context.Provider
       value={{
         web3: web3,
+        polygonClient: polygonClient,
```

Comment:
As @Yhtiyar pointed out it might be a good idea to fallback to user's metamask connection to polygon (if they have one) in case https://polygon-rpc.com goes down. Not necessary to implement on this PR.

Reply:
Adding this to the CU player portal updates.